tests/src/test/kotlin/dev/restate/e2e/WorkflowAPITest.kt

```diff
@@ -76,6 +77,7 @@ class NodeWorkflowAPITest {
   @Test
   @DisplayName("Set and resolve durable promise leads to completion")
   @Execution(ExecutionMode.CONCURRENT)
+  @Disabled("While suspension tests are resolved")
```

Comment:
Just FYI, you could also just disable the suspension test scenario by removing the annotation `@Tag("always-suspending")` on the `NodeWorkflowAPITest` class